notification.c

```diff
@@ -305,6 +327,9 @@ void notification_execute_binding(struct mako_notification *notif,
 	case MAKO_BINDING_DISMISS:
 		close_notification(notif, MAKO_NOTIFICATION_CLOSE_DISMISSED);
 		break;
+	case MAKO_BINDING_DISMISS_GROUP:
+		close_group_notifications(notif, MAKO_NOTIFICATION_CLOSE_DISMISSED);
```

Comment:
Maybe this function should take `notif->state` instead of `notif`.

Reply:
I think I need notif in order to create the criteria?

Reply:
Ah, indeed!